apis/app/v1beta1/kogitosupportingservice_types.go

```diff
@@ -38,6 +38,11 @@ func (k *KogitoSupportingServiceSpec) GetRuntime() api.RuntimeType {
 	return api.QuarkusRuntimeType
 }
 
+// IsRouteEnabled ...
+func (k *KogitoSupportingServiceSpec) IsRouteEnabled() bool {
+	return true
```

Comment:
We should allow the user to disable that route if needed. They may want to manage their own route system.

Reply:
In this case should the`spec.route` boolean flag control both Supporting Service routes and the normal KR routes?

Reply:
kogitosupportingservice should have https://github.com/kiegroup/kogito-operator/pull/1033/files#diff-5612b3ae7d9c41e756f09eb33d1fe50cb6e42d04380fd3fa0dd937a3f6f851dcR47 or maybe a `disableRoute` or similar which would be taken in that method

Reply:
Yes, I can add the field to [KogitoServiceSpec struct](https://github.com/kiegroup/kogito-operator/blob/178e5d4b83278e6540123a55ca9f745f0cc14bc1/apis/app/v1beta1/kogitoservices.go#L164), so the field exists for both KR spec and Supporting Service spec. If that flag is set to True in KR CR, then routes are configured for both KR and Supporting services; one field in CR controls both route configurations. 
Or, the other solution is to have 2 different fields in the CR, one controlling the SupportingService Routes and one controlling the regular KR routes. The first option seems cleaner, but it depends on how much flexibility we want to give the user.

Reply:
well I would prefer flexibility here. Service routes should not depend on KR anyway.
For me, **when KOGITO-1940 is implemented**, we could have:
- Kogito Runtime -> Route is disabled by default and can be enabled by the user
- Kogito Supporting Service -> Route is enabled by default and can be disabled by the user